sdk/eventhub/azure-eventhubs-aio-partitionmanager-blobstorage/azure/eventhub/aio/partitionmanager/blobstorage/__init__.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
I would consider removing this file/folder since there is only a single file in the package.